omegatrade-app/backend/app/models/user.model.js

```diff
@@ -0,0 +1,51 @@
+'user strict';
+const database = require('./../config/database.js');
+var User = function () {};
+
+User.registerUser = async function (user, cb) {
+  database.runTransaction(async (err, transaction) => {
+    if (err) {
+      cb(err, null)
+      return;
+    }
+    try {
+      const [rowCount] = await transaction.runUpdate({
+        sql: 'INSERT users (userId, fullName, businessEmail,password,photoUrl,provider) VALUES (@userId, @fullName, @businessEmail,@password,@photoUrl,@provider)',
+        params: {
+          userId: user.userId,
+          fullName: user.fullName,
+          businessEmail: user.businessEmail,
+          password: user.password,
+          photoUrl: user.photoUrl,
+          provider: user.provider
```

Comment:
why all properties are separately assigned? can't we pass use object directly here?